sweetest/sweetest-api/src/main/java/com/mysugr/sweetest/framework/cucumber/HooksSteps.kt

```diff
@@ -0,0 +1,54 @@
+package com.mysugr.sweetest.framework.cucumber
+
+import com.mysugr.sweetest.framework.context.TestContext
+import com.mysugr.sweetest.framework.flow.InitializationStep
+import com.mysugr.sweetest.usecases.proceedWorkflow
+import cucumber.api.java.After
```

Comment:
I think it's not very clean to have a cucumber dependency in `sweetest-api` that is then a transitive dependency that is always included.
Or is this cleaned up in a later PR?

Reply:
in the v1 API it will still be a transitive dependency in order to keep compatibility, but v2 will be freed of it, except if you use the `sweetest-cucumber2` artifact.